sdk/tables/azure-data-tables/azure/data/tables/_policies.py

```diff
@@ -353,19 +354,64 @@ def on_response(self, request, response):
                 )
 
 
-class StorageRetryPolicy(HTTPPolicy):
+class StorageRetryPolicy(RetryPolicy):
     """
-    The base class for Exponential and Linear retries containing shared code.
+    A base class for retry policies for the Table Client and Table Service Client
     """
+    def __init__(
+        self,
+        initial_backoff=15, # type: int
+        increment_base=3, # type: int
+        retry_total=10, # type: int
+        retry_to_secondary=False, # type: bool
+        random_jitter_range=3, # type: int
+        **kwargs # type: Any
+    ):
+        """
+        Build a BLANK retry object.
 
-    def __init__(self, **kwargs):
-        self.total_retries = kwargs.pop('retry_total', 10)
+        :param int initial_backoff:
+            The initial backoff interval, in seconds, for the first retry.
+        :param int increment_base:
+            The base, in seconds, to increment the initial_backoff by after the
+            first retry.
+        :param int max_attempts:
```

Comment:
not seeing `max_attepmts`in the init for this policy. Does this mean this sohuld be a keyword param, or is it something we're missing

Reply:
`max_attempts` and `retry_total` are synonymous in this situation, removing `max_attempts`